### PR TITLE
Sc refinement or fix

### PR DESCRIPTION
**Pull request by ysqyang**
# Description

Fixed bugs in OR policies

## Linked issue(s)/Pull request(s)

- [issue_number](issue_link)

## Type of Change

- [ ] Non-breaking bug fix
- [ ] Breaking bug fix
- [ ] New feature
- [ ] Test
- [ ] Doc update
- [ ] Docker update

## Related Component

- [ ] Simulation toolkit
- [ ] RL toolkit
- [ ] Distributed toolkit

## Has Been Tested

- OS:
  - [ ] Windows
  - [ ] Mac OS
  - [ ] Linux
- Python version:
  - [ ] 3.6
  - [ ] 3.7
- Key information snapshot(s):

## Needs Follow Up Actions

- [ ] New release package
- [ ] New docker image

## Checklist

- [ ] Add/update the related comments
- [ ] Add/update the related tests
- [ ] Add/update the related documentations
- [ ] Update the dependent downstream modules usage

### CODE REVIEW

File: examples/rl/supply_chain/policy_trainer.py [hunk 1]
```diff
@@ -3,39 +3,57 @@
...
+from .algorithms.rule_based import or_policy_creator
 from .config import rl_algorithm, NUM_CONSUMER_ACTIONS
 from .env_helper import entity_dict
 from .state_template import STATE_DIM
 
 
-def get_policy_name(entity_: SupplyChainEntity) -> str:
-    if entity_.is_facility:
+def get_policy_name(entity: SupplyChainEntity) -> str:
+    if entity.skus is None:
```
Comment (lihuoran) [inline examples/rl/supply_chain/policy_trainer.py:15]: Is `entity.skus is None` a good condition to recognize facilities?

File: examples/rl/supply_chain/policy_trainer.py [hunk 2]
```diff
@@ -3,39 +3,57 @@
...
+    policy_creator = {
+        policy_name: partial(get_policy, STATE_DIM, NUM_CONSUMER_ACTIONS)
+        for policy_name in agent2policy.values()
+    }
+    trainer_creator = {
+        policy_name.split(".")[0]: partial(get_ppo, STATE_DIM) for policy_name in agent2policy.values()
+    }
+elif rl_algorithm == 'null':
+    policy_creator = or_policy_creator
+    trainer_creator = None
```
Comment (lihuoran) [inline examples/rl/supply_chain/policy_trainer.py:57]: Is it better to use a empty dict here?

File: examples/rl/supply_chain/config.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+from os.path import dirname, join, realpath
```
Comment (lihuoran) [inline examples/rl/supply_chain/config.py:4]: Unused imports?

File: examples/rl/supply_chain/algorithms/rule_based.py [hunk 4]
```diff
@@ -112,28 +106,30 @@ def _rule(self, states: np.ndarray) -> np.ndarray:
...
 
 
-or_policy_func_dict = {
+or_policy_creator = {
+    "consumer_policy": lambda name: ConsumerBaselinePolicy(name),
     "manufacturer_policy": lambda name: ManufacturerBaselinePolicy(name),
     "facility_policy": lambda name: DummyPolicy(name),
     "product_policy": lambda name: DummyPolicy(name),
+    "seller_policy": lambda name: DummyPolicy(name)
 }  # TODO: never used
```
Comment (lihuoran) [inline examples/rl/supply_chain/algorithms/rule_based.py:135]: Remove this comment

File: examples/rl/supply_chain/callbacks.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import csv
+from os.path import dirname, join, realpath
+
+
+def post_collect(info_list: list, ep: int, segment: int) -> None:
+    with open(join(dirname(realpath(__file__)), "result_baseline.csv"), "a") as fp:
```
Comment (lihuoran) [inline examples/rl/supply_chain/callbacks.py:9]: Hardcoded file path?

File: examples/rl/supply_chain/policy_trainer.py (hunk 1 above)
Comment (ysqyang) [reply examples/rl/supply_chain/policy_trainer.py:15]: My original base branch did not have ``is_facility`` attribute for ``SupplyChainEntity``. Changed now.

File: examples/baseline.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation.
```
Comment (Jinyu-W) [inline examples/baseline.py:1]: move this file to examples/supply_chain/or_baseline.py ?

**Pull request merged**